packages/flutter/lib/src/widgets/image.dart

```diff
@@ -132,6 +132,9 @@ Future<void> precacheImage(
 ///  * [new Ink.image], which is the preferred way to show an image in a
 ///    material application (especially if the image is in a [Material] and will
 ///    have an [InkWell] on top of it).
+///  * [dart:ui.Image], the class in the [dart:ui] library that is
+///    encapsulated by this class and its subclasses.
```

Comment:
Image also doesn't have subclasses. Maybe you mean something else and I am misunderstanding this?